.golangci.yml

```diff
@@ -6,7 +6,6 @@ run:
 linters:
   disable-all: true
   enable:
-    - depguard
```

Comment:
Removed this linter because it has a ton of warnings. To resolve I think we need to define every library that is acceptable to import in this repo.

The `main` branch of Cosmos SDK no longer uses this linter and I think it wasn't actually working on the previous golangci-lint version